selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -395,6 +398,19 @@ public void clickOnCloseFileIcon(String fileName) {
         .click();
   }
 
+  /**
+   * check any tab is open in the Editor
+   *
+   * @return true if any tab is open
+   */
+  public boolean anyTabIsOpen() {
```

Comment:
`anyTabIsOpen` name is grammatically incorrect.
Better variant: `isAnyTabsOpened` (http://esl.fis.edu/grammar/rules/some.htm)